webrtc.html

```diff
@@ -83,6 +83,9 @@ <h2>Terminology</h2>
     <p>The terms <dfn>event</dfn>, <dfn data-cite="!HTML51/webappapis.html#events-event-handlers">event
     handlers</dfn> and <dfn data-cite="!HTML51/webappapis.html#event-handler-event-type">event
     handler event types</dfn> are defined in [[!HTML51]].</p>
+    <code><dfn data-cite="!HIGHRES-TIME#dom-performance-timeOrigin">performance.timeOrigin</dfn></code>
```

Comment:
Format nit: This one needs to be wrapped in <p> ... </p>, or formatting may be odd.